src/test/java/ee/buerokratt/ruuter/domain/steps/http/HttpGetStepTest.java

```diff
@@ -91,6 +94,16 @@ void execute_shouldExecuteDefaultActionWhenRequestIsInvalidAndStopInCaseOfExcept
         when(ci.getConfigurationService()).thenReturn(configurationService);
         when(ci.getMappingHelper()).thenReturn(mappingHelper);
         when(ci.getContext()).thenReturn(testContext);
+        when(ci.getHttpHelper().makeHttpGetRequest(expectedGetArgs)).thenReturn(httpResponse);
+        when(ci.getConfigurationService()).thenReturn(configurationService);
+        when(applicationProperties.getDefaultServiceInCaseOfException()).thenReturn(httpDefaultService);
+        doCallRealMethod().when(httpDefaultService).executeHttpDefaultAction(eq(ci), anyString());
+        when(ci.getRequestOrigin()).thenReturn("");
+        when(applicationProperties.getHttpCodesAllowList()).thenReturn(new ArrayList<>() {{add(200);}});
```

Comment:
Maybe this is a bit overkill, but i think it would look nicer if instead of `200` you used `HttpStatus.OK`. The difference is that 200 is an `int`, but `HttpStatus.OK` is status Object. It doesn't really matter in terms of the test, but generally i think it would look nicer.

Reply:
httpCodesAllowList is a list of Integers, in that case we would have to change that as well. Is that the goal here?